packages/sanity/src/_internal/cli/commands/migration/createMigrationCommand.ts

```diff
@@ -76,12 +76,12 @@ const createMigrationCommand: CliCommandDefinition<CreateMigrationFlags> = {
       .replace(/\s+/g, '-')
       .replace(/[^a-z0-9-]/g, '')
 
-    const destDir = path.join(MIGRATIONS_DIRECTORY, sluggedName)
+    const destDir = path.join(workDir, MIGRATIONS_DIRECTORY, sluggedName)
```

Comment:
In the spirit of being more prescriptive about where Sanity files and folders are located, I think this is a good change!

Reply:
Heck yeah!